v3/integration/config.json

```diff
@@ -638,47 +647,47 @@
     },
     "n_multiple_subject_rdn": {},
     "n_san_dns_name_duplicate": {
-      "NoticeCount": 193
+      "NoticeCount": 705
     },
     "n_san_iana_pub_suffix_empty": {
       "NoticeCount": 34
     },
     "n_sub_ca_eku_missing": {
-      "NoticeCount": 166
+      "NoticeCount": 678
     },
     "n_sub_ca_eku_not_technically_constrained": {
       "NoticeCount": 10
     },
     "n_subject_common_name_included": {
-      "NoticeCount": 44
+      "NoticeCount": 593196
```

Comment:
This...is especially terrifying.

Reply:
but expected! That's why it's a Notice level. The deprecation of subject CNs is... long and ongoing. Honestly seeing only 44 notices in our corpus before for this lint should have been a red flag :sweat: